docs/resources/confluent_catalog_entity_attributes.md

```diff
@@ -23,7 +23,51 @@ provider "confluent" {
   cloud_api_secret = var.confluent_cloud_api_secret # optionally use CONFLUENT_CLOUD_API_SECRET env var
 }
 
-resource "confluent_catalog_entity_attributes" "main" {
+resource "confluent_catalog_entity_attributes" "environment" {
```

Comment:
Also I saw some usage of `confluent_catalog_entity_attributes.main` in the argument reference section, I guess it's `.environmnet` now?

Reply:
I just created APIT-2477 🫡